src/LeagueToolkit/Utils/Extensions/StreamExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace LeagueToolkit.Utils.Extensions
```

Comment:
Please change to file-scoped namespace

Reply:
This should be configured in the editorconfig accordingly then:
```diff
diff --git a/src/LeagueToolkit/.editorconfig b/src/LeagueToolkit/.editorconfig
index 7eb46fe..c9dc7bb 100644
--- a/src/LeagueToolkit/.editorconfig
+++ b/src/LeagueToolkit/.editorconfig
@@ -109,7 +109,7 @@ csharp_style_prefer_readonly_struct = true
 # Code-block preferences
 csharp_prefer_braces = true
 csharp_prefer_simple_using_statement = true
-csharp_style_namespace_declarations = block_scoped
+csharp_style_namespace_declarations = file_scoped
 csharp_style_prefer_method_group_conversion = true
 csharp_style_prefer_top_level_statements = true

```